components/Input/Input.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import { Meta, StoryFn } from "@storybook/react";
+import Input from "./Input";
+
+export default {
+  title: "Input",
+  component: Input,
+} as Meta<typeof Input>;
+
+export const Default: StoryFn<typeof Input> = () => (
+  <Input label="Texto" helperText="Mensaje error" name="input" />
+);
```

Comment:
Actualmente Storybook no nos esta permitiendo "interactuar" con las historias, para habilitar esto puedes pasarle los `args` a las historias de esta forma:
> Puedes ver mas ejemplos en [este link](https://storybook.js.org/docs/react/essentials/controls)

```suggestion
export const Default: StoryFn<typeof Input> = (args) => (
  <Input {...args} label="Texto" helperText="Mensaje error" name="input" />
);
```